Dockerfile

```diff
@@ -20,4 +20,5 @@ FROM debian:bullseye-slim as run
 RUN apt-get update && apt-get -y install ca-certificates libc6
 COPY --from=build /app/target/release/lite-rpc /usr/local/bin/
 
+CMD ethtool -k DEVICE | grep generic-segmentation-offload
```

Comment:
DEVICE should be replace with the device name.
Add lines to enable it again and then repeat this line to check if it enabled.